src/js/view/mxGraph.js

```diff
@@ -4607,6 +4607,31 @@ mxGraph.prototype.insertEdge = function(parent, id, value, source, target, style
 	return this.addEdge(edge, parent, source, target);
 };
 
+/**
+ * Function: insertInvertedEdge
+ * 
+ * Adds a new edge into the given parent <mxCell> using value as the user
+ * object and the given source and target as the terminals of the new edge.
+ * The id and style are used for the respective properties of the new
+ * <mxCell>, which is returned.
+ *
+ * Parameters:
+ * 
+ * parent - <mxCell> that specifies the parent of the new edge.
+ * id - Optional string that defines the Id of the new edge.
+ * value - JavaScript object to be used as the user object.
+ * source - <mxCell> that defines the source of the edge.
+ * target - <mxCell> that defines the target of the edge.
+ * style - Optional string that defines the cell style.
+ */
+ mxGraph.prototype.insertInvertedEdge = function(parent, id, value, source, target, style)
```

Comment:
do think in the longer-term it would be better to create a derived class such as `azdataGraph` and add our customizations there?  or does it not really matter if we start modifying the base `mxGraph` classes?  Since we're not taking any additional code refreshes from upstream I think it probably doesn't matter.  But if we isolated our changes it may make debugging, testing, etc. more scoped since we can assume the upstream source is basically already tested.